jmx-metrics/build.gradle

```diff
@@ -42,7 +42,7 @@ def deps = [
 repositories {
     mavenCentral()
     maven {
-        setUrl("http://www.terracotta.org/download/reflector/releases")
+        setUrl("https://repo.terracotta.org/maven2")
```

Comment:
This isn't documented on terracotta's site but seems to be the only way to use HTTPS. It seemed to have worked